src/main/java/org/baeldung/security/LoggedUsers.java

```diff
@@ -0,0 +1,30 @@
+package org.baeldung.security;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LoggedUsers {
+
+    private static LoggedUsers instance = null;
+
+    public static List<String> users;
+
+    private LoggedUsers() {
+        users = new ArrayList<String>();
+    }
+
+    public static LoggedUsers getInstance() {
+        if (instance == null) {
+            instance = new LoggedUsers();
+        }
+        return instance;
+    }
+
+    public static List<String> getUsers() {
+        return users;
+    }
+
+    public static void setUsers(List<String> users) {
+        LoggedUsers.users = users;
+    }
+}
```

Comment:
There is a better way to use singleton instance in spring. You can create bean with default scope in AppConfig.java as:

@Bean
public ActiveUserStore activeUserStore() {
    return new ActiveUserStore();
}

Then you can just define ActiveUserStore as plain java class without any static function.

To use it in other classes, you can just do
@Autowired
ActiveUserStore activeUserStore;